File: app/src/main/java/com/qwe7002/telegram_sms/wap_receiver.java
package com.qwe7002.telegram_sms;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;
import android.util.Log;

import java.util.Objects;

public class wap_receiver extends BroadcastReceiver {
    @Override
    public void onReceive(final Context context, Intent intent) {
        Log.d("wap_receiver", "Receive action: "+intent.getAction());
    }
}

File: app/src/main/java/com/airfreshener/telegram_sms/WapReceiver.kt
package com.airfreshener.telegram_sms

import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent
import android.util.Log

class WapReceiver : BroadcastReceiver() {
    override fun onReceive(context: Context, intent: Intent) {
        Log.d("WapReceiver", "Receive action: " + intent.action)
    }
}
